examples/pending-order-reminder/Send Reminder on Approval Pending Orders.md

```diff
@@ -0,0 +1,35 @@
+# Send reminder on approval pending sales orders
```

Comment:
```suggestion
# Send a reminder on approval of pending sales orders
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-sap/pull/10